src/extension.ts

```diff
@@ -1737,6 +1738,30 @@ const debugAttachKubernetes = async (explorerNode: ClusterExplorerResourceNode)
     }
 };
 
+const selectPodKubernetes = async () => {
+    const resource =  await kubectlUtils.getResourceAsJson<KubernetesCollection<Pod>>(kubectl, "pods");
+    if (!resource) {
+        return;
+    }
+
+    const podPickItems = resource.items.map((pod) => {
+        return {
+            label: `${pod.metadata.name} (${pod.spec.nodeName})`,
+            description: "pod",
```

Comment:
Is there a reason why you also add namespace, containers, description properties? I guess this could be simplified as 
```
const podPickItems = resource.items.map((pod) => {
        return {
            label: `${pod.metadata.name} (${pod.spec.nodeName})`,
            name: pod.metadata.name
        };
    });
```

Reply:
That's a good point. I removed the unused properties.